core/modules/set_metadata.cc

```diff
@@ -71,6 +134,8 @@ CommandResponse SetMetadata::AddAttrOne(
   size_t size = 0;
   int offset = -1;
   value_t value = value_t();
+  mask_t mask = mask_t();
```

Comment:
I am not entirely sure, but the default constructor of `mask_t` is no-op, thus this definition is not different from just `mask_t mask`?